app/views/waste_calendar/new.html.erb

```diff
@@ -0,0 +1,58 @@
+<div class="d-sm-flex align-items-center justify-content-between">
+  <h1 class="h3 mb-0 text-gray-800">Daten für Abfallkalender importieren</h1>
```

Comment:
wenn das `create` "Schritt 2:" ist, dann ist das hier "Schritt 1:", richtig? füge ich hinzu

Reply:
done in 8def4ab5bbc68a9c690785024135d820842578c3